python/tests/test_snippets.py

```diff
@@ -116,6 +116,67 @@ def test_code_snippet(code: etree.ElementTree) -> None:
                 'structureTypeFrench': 'Ossature de bois',
                 'componentTypeSizeEnglish': '38x89 mm (2x4 in)',
                 'componentTypeSizeFrench': '38x89 (2x4)',
-            }]
+            }],
+            'window': [{
```

Comment:
This test is getting a bit long. Maybe break it up into separate tests, and not necessarily testing the whole output.